File: Leetcode-problems/twosum.js
var twoSum = function(nums, target) {
    for (let i = 0; i <nums.length; i++) {
            for (let j = 0; j < nums.length; j++) {
                if (i != j) {
                    let result = nums[i] + nums[j];
                    if (result == target) {
                        let newArray = [i, j];
                        return newArray;
                    }
                }
            }
        }
};

File: C-Sharp-Leetcode/twosum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for (int i = 0; i <nums.Length; i++)
        {
            for (int j = 0; j < nums.Length; j++)
            {
                if (i != j)
                {
                    int result = nums[i] + nums[j];
                    if (result == target)
                    {
                        return new int[] {i, j};
                    }
                }
            }
        }
        return null;
    }
}